src/Controller/ProductListItemView.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Sylius\ElasticSearchPlugin\Controller;
+
+final class ProductListItemView
```

Comment:
Maybe we should put this files in a separate namespace? Also, does it make sense to use views from ShopApiPlugin?